doc/source/file_formats.rst

```diff
@@ -46,9 +46,54 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+.. note::
+    The following information about SWC format concern the standard representation as 
+    it has been established in the neuromorpho.org database. We would like to clarify
+    that this file format is not yet supported by NeuroM, but it will be modified 
+    to do so, in the future.
+
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
+The soma can be represented in one of the following formats: 
+
+TypeA: One point soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by its center and a radius that corresponds to the sphere
+that preserves the surface area of the soma. The center of the soma has as parent ID -1 
+and all the initial points of the neuronal trees are connected to the soma. 
+
+TypeB: Three point soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by three points that correspond to the center of the soma (x, y, z), 
+with parent ID -1, and two diametrically opposite points (x, y-r, z) and (x, y+r, z), where r 
+is the radius of the sphere that  preserves the surface area of the soma. The radii of the soma 
+points are not necessarily meaningful, so they can be set to zero. As such, they should not be 
+taken into account in further calculations. The other two of the soma points are connected to the 
+center of the soma, as well as the first points of all the neuronal trees. 
+
+TypeC: N - points soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by a set of points. The first point corresponds to the center of the soma 
```

Comment:
Probably it should not be, but this is the case for most of the data as I understand it. This has no effect on the computation of the actual center but has a serious effect on the computation of the soma radius. Ideally, we should have a way of knowing if the first point is the center or not.